tests/ui/let_if_seq.stderr

```diff
@@ -1,3 +1,23 @@
+error: unneeded late initalization
```

Comment:
This warning is confusing in `leg_if_seq` test file, so can you add `#[allow(clippy::needless_late_init)]`?